### DISCUSSION TITLE
When will be the next release?

### DISCUSSION THREAD

**Discussion by graue70** [blueman-project/blueman#2264] [Q&A]
A fix to my problem has been in main for more than a year but there hasn't been a release since Nov 2022.

1. Why?
2. When will be the next release?

**Comment by cschramm** [comment]
As it should be obvious that nobody here will give any estimates or justifications for his spare time and the part of it that he spends on blueman, I'll try to rephrase the questions to something more constructive:

> 1. What's missing and how can I help with it?

Easy. Check [the milestone](https://github.com/blueman-project/blueman/milestone/13) and look into the missing bits to get the state release ready. As you can see, we postponed many things to a hypothetical 2.5 milestone to get to a release and it's close. If you think you can help with the missing bits, just dive right into it. Otherwise there's actually not much you can do except sponsoring the project (that will not magically generate time, though, and you would be the very first sponsor to say the least) and refrain from asking such questions. A pretty huge share of the time that I manage to spend on blueman gets taken by dealing with users, often in a first level Linux Mint support fashion, instead of something productive.

> 2. How can I get a specific version of blueman?

If you're a distributor, feel free to pick any version you want. We recommend releases, but obviously you do not need one. If you're a user, please ask your distributor. You can usually build your own packages and there are git snapshot packages e.g. for Arch Linux and Gentoo Linux and I maintain some for Debian-based distributions [here](https://blueman.cschramm.eu/debian/). Note that Debian-based distributions typically do not have a concept of feature updates for their releases. They do have the software they have and it's usually not very recent, especially if they are based on something like Ubuntu LTS.
